app/webpacker/components/Live/Admin/DoubleCheck/index.jsx

```diff
@@ -0,0 +1,151 @@
+    });
+  };
+
+  const handleAttemptChange = (index, value) => {
+    const newAttempts = [...attempts];
+    newAttempts[index] = value;
+    setAttempts(newAttempts);
+  };
+
+  const handleRegistrationIdChange = useCallback((_, { value }) => {
+    setRegistrationId(value);
+    const alreadyEnteredResults = results.find((r) => r.registration_id === value);
+    setAttempts(alreadyEnteredResults.attempts.map((a) => a.result));
+  }, [results]);
+
+  const currentCompetitor = useMemo(() => {
+    if (results) {
+      return competitors.find((r) => r.id === results[currentIndex].registration_id);
+    }
+    return {};
+  }, [competitors, currentIndex, results]);
+
+  useEffect(() => {
+    if (results) {
+      setAttempts(results[currentIndex].attempts.map((a) => a.result));
+    }
+  }, [currentIndex, results]);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  return (
+    <Grid>
+      <Grid.Column width={1} verticalAlign="middle">
+        { currentIndex !== 0 && <Button onClick={() => setCurrentIndex((oldIndex) => oldIndex - 1)}>{'<'}</Button>}
+      </Grid.Column>
+      <Grid.Column width={7}>
+        <Segment loading={isPendingUpdate}>
+          <Form>
+            <Form.Select
+              label="Competitor"
+              placeholder="Competitor"
+              value={currentCompetitor.id}
+              search={(inputs, value) => inputs.filter((d) => d.text.toLowerCase().includes(value.toLowerCase()) || parseInt(value, 10) === d.registrationId)}
+              onChange={handleRegistrationIdChange}
+              options={competitors.map((p) => ({
+                key: p.id,
+                value: p.id,
+                registrationId: p.registration_id,
+                text: `${p.user.name} (${p.registration_id})`,
+              }))}
+            />
+            {Array.from(zeroedArrayOfSize(solveCount).keys()).map((index) => (
+              <AttemptResultField
+                eventId={eventId}
+                key={index}
+                label={`Attempt ${index + 1}`}
+                placeholder="Time in milliseconds or DNF"
+                value={attempts[index] ?? 0}
+                onChange={(value) => handleAttemptChange(index, value)}
+              />
+            ))}
+            <Form.Button primary onClick={handleSubmit}>Submit Results</Form.Button>
+          </Form>
+        </Segment>
+      </Grid.Column>
+      <Grid.Column width={1} verticalAlign="middle">
+        {currentIndex !== results.length - 1 && <Button onClick={() => setCurrentIndex((oldIndex) => oldIndex + 1)}>{'>'}</Button>}
+      </Grid.Column>
+      <Grid.Column textAlign="center" width={7} verticalAlign="middle">
+        <Card fluid raised>
+          <Card.Header>
+            {currentIndex + 1}
+            {' '}
+            of
+            {' '}
+            {results.length}
+          </Card.Header>
+          <Card.Content>
+            <Header>{round.name}</Header>
```

Comment:
You just completed a `Card.Header` above and the first thing you're doing in the body is opening a `Header`. Why not just move `round.name` into the card header?

Reply:
done, but it does need a break tag currently. This can be fixed when the "x of y" string will be in i18n